src/System.Drawing.Common/src/System.Drawing.Common.csproj

```diff
@@ -9,7 +9,7 @@
     <!-- TODO: RE-ENABLE THIS -->
     <EnablePInvokeAnalyzer>false</EnablePInvokeAnalyzer>
     <WarningsNotAsErrors>CS0618</WarningsNotAsErrors>
-    <DefineConstants Condition="'$(TargetsWindows)' == 'true'">$(DefineConstants);FEATURE_WINDOWS_SYSTEM_COLORS</DefineConstants>
+    <DefineConstants Condition="'$(TargetsWindows)' == 'true'">$(DefineConstants);FEATURE_WINDOWS_SYSTEM_COLORS;NETCORE</DefineConstants>
```

Comment:
In this project TargetsWindows not necessarily means NETCORE only, it could also be netfx facade and this project is build for unix as netcoreapp.

Reply:
Maybe NETCORE_WINDOWS ? Or FEATURE_FONT_WINDOWS?